app/routes/workshops/$slug.tsx

```diff
@@ -32,34 +35,333 @@ export default function MdxScreenBase() {
+                <H2 className="mb-16" variant="secondary" as="p">
+                  Here’s why you should register for the workshop.
+                </H2>
+                <Button>Register here</Button>
+              </div>
+
+              <div className="col-span-full lg:col-span-5 lg:col-start-8 lg:mr-12">
+                <ol className="space-y-24 lg:space-y-16">
+                  {Array.from({length: 3}).map((_, idx) => (
+                    <NumberedPanel
+                      key={idx}
+                      number={idx + 1}
+                      caption="Here will a random long title that doesn't fit on one line."
+                      description="Praesent eu lacus odio. Pellentesque vitae lectus tortor. Donec elit
+        nunc, dictum quis condimentum in, imper diet at arcu."
+                    />
+                  ))}
+                </ol>
+              </div>
+            </Grid>
+          </div>
+        </div>
+      </div>
+
+      <Grid className="mb-48 lg:mb-64">
+        <div className="col-span-8 mb-8 lg:mb-16">
+          <H2 className="mb-4 lg:mb-2">The topics we will be covering.</H2>
+          <H2 variant="secondary">This is what we will talk about.</H2>
+        </div>
+
+        <div className="flex flex-col col-span-full items-stretch justify-end mb-16 lg:col-span-4 lg:items-end lg:justify-center">
+          <Button>register here</Button>
+        </div>
+
+        <ol className="col-span-full mb-16 space-y-4 lg:mb-20">
+          {Array.from({length: 4}).map((_, idx) => (
+            <TopicRow
+              key={idx}
+              number={idx + 1}
+              topic="Use the Compound Components Pattern to write React components
+        that implicitly share state while giving rendering flexibility
+        to the user."
+            />
+          ))}
+        </ol>
+
+        <div className="col-span-full lg:col-span-5">
+          <H6 className="mb-4">Required experience</H6>
+          <Paragraph>
+            Attend my Advanced React Hooks Workshop or have the equivalent
+            experience. You should be experienced with useContext and useReducer
+            (experience with useMemo and useCallback is a bonus).
+          </Paragraph>
+        </div>
+      </Grid>
+
+      <Grid>
+        <div className="col-span-full mb-12 lg:col-span-8 lg:mb-16">
+          <H2 className="mb-2">Don’t just take my word for it.</H2>
+          <H2 variant="secondary" as="p">
+            What participants have to say.
+          </H2>
+        </div>
+        {/*
+          TODO: connect these buttons to make the testimonials scroll,
+           I think we need to use some js to get the testimonials width,
+           and then apply a transformX using framer.motion
+         */}
+        <div className="hidden col-span-2 col-start-11 items-end justify-end mb-16 space-x-3 lg:flex">
+          <ArrowButton direction="left" />
+          <ArrowButton direction="right" />
+        </div>
```

Comment:
This will probably take some time, and as this page already took more than I hoped for, I propose to fix that via another PR. (when we're going to focus on other motion stuff as well)